### PR TITLE
Common: added lateral latency parameters to vehicle config proto

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/common/configs/proto/vehicle_config.proto [hunk 1]
```diff
@@ -82,6 +82,12 @@ message VehicleParam {
   optional double brake_deadzone = 20 [default = nan];
   // minimum value get from chassis.throttle, in percentage
   optional double throttle_deadzone = 21 [default = nan];
+
+  //latency parameters, in second
+  optional double steering_dead_time = 22 [default = 0.0];
+  optional double steering_peak_time = 23 [default = 0.0];
+  optional double steering_rise_time = 24 [default = 0.0];
```
Comment (kevin-y-wang) [inline modules/common/configs/proto/vehicle_config.proto]: Small suggestion: could we switch the order of peak_time and rise_time?

File: modules/common/configs/proto/vehicle_config.proto [hunk 2]
```diff
@@ -82,6 +82,12 @@ message VehicleParam {
   optional double brake_deadzone = 20 [default = nan];
   // minimum value get from chassis.throttle, in percentage
   optional double throttle_deadzone = 21 [default = nan];
+
+  //latency parameters, in second
+  optional double steering_dead_time = 22 [default = 0.0];
+  optional double steering_rise_time = 23 [default = 0.0];
+  optional double steering_peak_time = 24 [default = 0.0];
+  optional double steering_settling_time = 25 [default = 0.0];
```
Comment (xiaoxq) [inline modules/common/configs/proto/vehicle_config.proto]: Small suggestion, you don't need to specify `default = 0` for any numeric types, as it's the default default.

File: modules/common/configs/proto/vehicle_config.proto [hunk 3]
```diff
@@ -82,10 +82,23 @@ message VehicleParam {
...
 }
 
 message VehicleConfig {
   optional apollo.common.Header header = 1;
   optional VehicleParam vehicle_param = 2;
   optional Extrinsics extrinsics = 3;
 }
+
+message LatencyParam {
+    //latency parameters, in microsecond (ms)
```
Comment (xiaoxq) [inline modules/common/configs/proto/vehicle_config.proto]: Space after `//`

File: modules/common/configs/proto/vehicle_config.proto [hunk 4]
```diff
@@ -42,6 +42,14 @@ message VehicleID {
   optional string other_unique_id = 3;
 }
 
+message LatencyParam {
+    // latency parameters, in microsecond (ms)
+    optional double dead_time = 1;
+    optional double rise_time = 2;
+    optional double peak_time = 3;
+    optional double settling_time = 4;
```
Comment (xiaoxq) [inline modules/common/configs/proto/vehicle_config.proto]: Unify indents to be the same within the file.

**Pull request merged**